client/src/lib/components/RoomForm.svelte

```
@@ -82,74 +82,79 @@
   }
 </script>
 
-<div>
-  <form>
-    <label for="roomName">Room</label>
-    <input
-      type="text"
-      id="roomName"
-      bind:value={roomName}
-      placeholder="Enter Room Name"
-      required
-    />
-    <div class="error-text">{roomNameError}</div>
-    <label for="roomName">Name</label>
-    <input
-      type="text"
-      bind:value={playerName}
-      placeholder="Enter Your Player Name"
-      required
-    />
-    <div class="error-text">{playerNameError}</div>
-    <button
-      id="joinRoom"
-      type="submit"
-      onclick={joinRoom}
-      disabled={!inputsFilled}
-      class="mt-2 bg-blue-500 text-white p-2 rounded w-full"
-    >
-      Join Room
-    </button>
-    <button
-      id="createRoom"
-      type="submit"
-      onclick={createRoom}
-      disabled={!inputsFilled}
-      class="mt-2 bg-blue-500 text-white p-2 rounded w-full"
-    >
-      Create Room
-    </button>
+<div class="flex-center-page">
+  <form class="paper-card room-form">
+    <div class="form-group">
+      <label for="roomName" class="text-pen-blue">Room</label>
+      <input
+        type="text"
+        id="roomName"
+        bind:value={roomName}
+        placeholder="Enter Room Name"
+        required
+      />
+      <div class="text-error">{roomNameError}</div>
+    </div>
+    <div class="form-group">
+      <label for="playerName" class="text-pen-blue">Name</label>
+      <input
+        type="text"
+        id="playerName"
+        bind:value={playerName}
+        placeholder="Enter Your Player Name"
+        required
+      />
+      <div class="text-error">{playerNameError}</div>
+    </div>
+    <div class="button-group">
+      <button
+        id="joinRoom"
+        type="submit"
+        onclick={joinRoom}
+        disabled={!inputsFilled}
+      >
+        Join Room
```

Currently there's no error messages the user can see when they try to create a room, it just fails silently. Can be something I can address over break. EDIT: I lied it just took a long time, so maybe we figure out why the error took so long to display